unfortunately this doesn't seem to agree with Windows
```
D:\a\Shipwright\Shipwright\soh\soh\resource\importer\AudioSampleFactory.cpp(130): error C2956: usual deallocation function 'void operator delete[](void *,std::align_val_t) noexcept' would be chosen as placement deallocation function.
predefined C++ types (compiler internal)(72): note: see declaration of 'operator delete[]'
```

--- a/soh/soh/Enhancements/Restorations/GraveHoleJumps.cpp
+++ b/soh/soh/Enhancements/Restorations/GraveHoleJumps.cpp
@@ -47,7 +47,7 @@ CollisionHeader* getGraveyardCollisionHeader() {
      * into graves. NTSC 1.0's graveyard has 31 surface types, while later versions have 32. The contents of the lists
      * are shifted somewhat between versions, so to be safe we just create an extra slot that is not in any version.
      */
-    static SurfaceType newSurfaceTypes[33] = {};
+    static SurfaceType newSurfaceTypes[33];
     memcpy(newSurfaceTypes, graveyardColHeader->surfaceTypeList, sizeof(SurfaceType) * surfaceTypesCount);
     newSurfaceTypes[CUSTOM_SURFACE_TYPE].data[0] = 0x24000004;
     newSurfaceTypes[CUSTOM_SURFACE_TYPE].data[1] = 0xFC8;

